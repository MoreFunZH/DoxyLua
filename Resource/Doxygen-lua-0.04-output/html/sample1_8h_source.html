<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.6"/>
<title>Sample Project: sample1.h Source File</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="navtree.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="resize.js"></script>
<script type="text/javascript" src="navtree.js"></script>
<script type="text/javascript">
  $(document).ready(initResizable);
  $(window).load(resizeHeight);
</script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">Sample Project
   </div>
   <div id="projectbrief">This is the sample output of DoxyLua.</div>
  </td>
   <td>        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
</td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.6 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
</div><!-- top -->
<div id="side-nav" class="ui-resizable side-nav-resizable">
  <div id="nav-tree">
    <div id="nav-tree-contents">
      <div id="nav-sync" class="sync"></div>
    </div>
  </div>
  <div id="splitbar" style="-moz-user-select:none;" 
       class="ui-resizable-handle">
  </div>
</div>
<script type="text/javascript">
$(document).ready(function(){initNavTree('sample1_8h_source.html','');});
</script>
<div id="doc-content">
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Variables</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="header">
  <div class="headertitle">
<div class="title">sample1.h</div>  </div>
</div><!--header-->
<div class="contents">
<a href="sample1_8h.html">Go to the documentation of this file.</a><div class="fragment"><div class="line"><a name="l00001"></a><span class="lineno">    1</span>&#160;</div>
<div class="line"><a name="l00006"></a><span class="lineno">    6</span>&#160;</div>
<div class="line"><a name="l00009"></a><span class="lineno"><a class="line" href="sample1_8h.html#a5b501f35575f30c22fdc725493e82034">    9</a></span>&#160;    SomeEnum.Enum1 = 1; </div>
<div class="line"><a name="l00010"></a><span class="lineno"><a class="line" href="sample1_8h.html#a1df5ca10faf13981dc8a501722ccff6a">   10</a></span>&#160;    SomeEnum.Enum2 = 2; </div>
<div class="line"><a name="l00011"></a><span class="lineno"><a class="line" href="sample1_8h.html#ae642968710e85cb476166498633933a8">   11</a></span>&#160;    SomeEnum.Enum3 = 3; </div>
<div class="line"><a name="l00012"></a><span class="lineno">   12</span>&#160;</div>
<div class="line"><a name="l00016"></a><span class="lineno">   16</span>&#160;</div>
<div class="line"><a name="l00017"></a><span class="lineno"><a class="line" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">   17</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a> = newClass();</div>
<div class="line"><a name="l00018"></a><span class="lineno">   18</span>&#160;</div>
<div class="line"><a name="l00021"></a><span class="lineno"><a class="line" href="sample1_8h.html#a1c9d3d4c43065088ecbdd3b621ed37ee">   21</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>.prop1 = nil;</div>
<div class="line"><a name="l00022"></a><span class="lineno">   22</span>&#160;</div>
<div class="line"><a name="l00023"></a><span class="lineno"><a class="line" href="sample1_8h.html#a8942bb2211994b84394ed5b547f2fc41">   23</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>.prop2 = nil; </div>
<div class="line"><a name="l00024"></a><span class="lineno">   24</span>&#160;</div>
<div class="line"><a name="l00030"></a><span class="lineno">   30</span>&#160;<span class="keyword">function</span> <a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>-<a class="code" href="sample1_8h.html#a2c4f24fca2a7086dce27836aceb79821">func1</a>(arg1, arg2);</div>
<div class="line"><a name="l00031"></a><span class="lineno">   31</span>&#160;</div>
<div class="line"><a name="l00037"></a><span class="lineno">   37</span>&#160;<span class="keyword">function</span> <a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>-<a class="code" href="sample1_8h.html#a6ed1116c1619bf1bbf68081b2bc4f08b">func2</a>(arg1, arg2);</div>
<div class="ttc" id="sample1_8h_html_a2c4f24fca2a7086dce27836aceb79821"><div class="ttname"><a href="sample1_8h.html#a2c4f24fca2a7086dce27836aceb79821">func1</a></div><div class="ttdeci">function CClass func1(arg1, arg2)</div><div class="ttdoc">This is CClass&#39;s member property prop2. </div></div>
<div class="ttc" id="sample1_8h_html_a6ed1116c1619bf1bbf68081b2bc4f08b"><div class="ttname"><a href="sample1_8h.html#a6ed1116c1619bf1bbf68081b2bc4f08b">func2</a></div><div class="ttdeci">function CClass func2(arg1, arg2)</div></div>
<div class="ttc" id="sample1_8h_html_a2b69c74036cc3a2a35c5aac054dfa154"><div class="ttname"><a href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a></div><div class="ttdeci">CClass</div><div class="ttdoc">This is Enum3. </div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00017">sample1.h:17</a></div></div>
</div><!-- fragment --></div><!-- contents -->
</div><!-- doc-content -->
<!-- start footer part -->
<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
  <ul>
    <li class="navelem"><a class="el" href="sample1_8h.html">sample1.h</a></li>
    <li class="footer">Generated on Sat Mar 29 2014 17:29:50 for Sample Project by
    <a href="http://www.doxygen.org/index.html">
    <img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.8.6 </li>
  </ul>
</div>
</body>
</html>
