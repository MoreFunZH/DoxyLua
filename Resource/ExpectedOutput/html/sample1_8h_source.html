<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.6"/>
<title>Sample Project: folder/sample1.h Source File</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="navtree.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="resize.js"></script>
<script type="text/javascript" src="navtree.js"></script>
<script type="text/javascript">
  $(document).ready(initResizable);
  $(window).load(resizeHeight);
</script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">Sample Project
   </div>
   <div id="projectbrief">This is the sample output of DoxyLua.</div>
  </td>
   <td>        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
</td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.6 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
</div><!-- top -->
<div id="side-nav" class="ui-resizable side-nav-resizable">
  <div id="nav-tree">
    <div id="nav-tree-contents">
      <div id="nav-sync" class="sync"></div>
    </div>
  </div>
  <div id="splitbar" style="-moz-user-select:none;" 
       class="ui-resizable-handle">
  </div>
</div>
<script type="text/javascript">
$(document).ready(function(){initNavTree('sample1_8h_source.html','');});
</script>
<div id="doc-content">
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Enumerations</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Enumerator</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="header">
  <div class="headertitle">
<div class="title">sample1.h</div>  </div>
</div><!--header-->
<div class="contents">
<a href="sample1_8h.html">Go to the documentation of this file.</a><div class="fragment"><div class="line"><a name="l00001"></a><span class="lineno">    1</span>&#160;</div>
<div class="line"><a name="l00007"></a><span class="lineno">    7</span>&#160;</div>
<div class="line"><a name="l00009"></a><span class="lineno"><a class="line" href="sample1_8h.html#afcac3511429181907b324fa566e6147d">    9</a></span>&#160;<span class="keyword">enum</span> <a class="code" href="sample1_8h.html#afcac3511429181907b324fa566e6147d">SomeEnum</a> {</div>
<div class="line"><a name="l00010"></a><span class="lineno"><a class="line" href="sample1_8h.html#afcac3511429181907b324fa566e6147da8ac106a2a0779565ad972ac4314f4d10">   10</a></span>&#160;  <a class="code" href="sample1_8h.html#afcac3511429181907b324fa566e6147da8ac106a2a0779565ad972ac4314f4d10">Enum1</a> = 1, </div>
<div class="line"><a name="l00011"></a><span class="lineno"><a class="line" href="sample1_8h.html#afcac3511429181907b324fa566e6147dab835515b078d0fde7ddfb245937aa0aa">   11</a></span>&#160;  <a class="code" href="sample1_8h.html#afcac3511429181907b324fa566e6147dab835515b078d0fde7ddfb245937aa0aa">Enum2</a> = 2, </div>
<div class="line"><a name="l00012"></a><span class="lineno"><a class="line" href="sample1_8h.html#afcac3511429181907b324fa566e6147da6d8bd6a7b94429614d045d1cb574d7ef">   12</a></span>&#160;  <a class="code" href="sample1_8h.html#afcac3511429181907b324fa566e6147da6d8bd6a7b94429614d045d1cb574d7ef">Enum3</a> = 3, </div>
<div class="line"><a name="l00013"></a><span class="lineno">   13</span>&#160;};</div>
<div class="line"><a name="l00014"></a><span class="lineno">   14</span>&#160;</div>
<div class="line"><a name="l00016"></a><span class="lineno"><a class="line" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">   16</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a> = newClass();</div>
<div class="line"><a name="l00017"></a><span class="lineno">   17</span>&#160;</div>
<div class="line"><a name="l00019"></a><span class="lineno"><a class="line" href="sample1_8h.html#a1c9d3d4c43065088ecbdd3b621ed37ee">   19</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>.prop1 = nil;</div>
<div class="line"><a name="l00020"></a><span class="lineno">   20</span>&#160;</div>
<div class="line"><a name="l00021"></a><span class="lineno"><a class="line" href="sample1_8h.html#a8942bb2211994b84394ed5b547f2fc41">   21</a></span>&#160;<a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a>.prop2 = nil; </div>
<div class="line"><a name="l00022"></a><span class="lineno">   22</span>&#160;</div>
<div class="line"><a name="l00028"></a><span class="lineno">   28</span>&#160;<span class="keyword">function</span> <a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a> <a class="code" href="sample1_8h.html#a2c4f24fca2a7086dce27836aceb79821">func1</a>(arg1, arg2);</div>
<div class="line"><a name="l00029"></a><span class="lineno">   29</span>&#160;</div>
<div class="line"><a name="l00035"></a><span class="lineno">   35</span>&#160; <span class="keyword">function</span> <a class="code" href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a> <a class="code" href="sample1_8h.html#a6ed1116c1619bf1bbf68081b2bc4f08b">func2</a>(arg1, arg2);</div>
<div class="line"><a name="l00036"></a><span class="lineno">   36</span>&#160;</div>
<div class="ttc" id="sample1_8h_html_a2c4f24fca2a7086dce27836aceb79821"><div class="ttname"><a href="sample1_8h.html#a2c4f24fca2a7086dce27836aceb79821">func1</a></div><div class="ttdeci">function CClass func1(arg1, arg2)</div></div>
<div class="ttc" id="sample1_8h_html_a6ed1116c1619bf1bbf68081b2bc4f08b"><div class="ttname"><a href="sample1_8h.html#a6ed1116c1619bf1bbf68081b2bc4f08b">func2</a></div><div class="ttdeci">function CClass func2(arg1, arg2)</div></div>
<div class="ttc" id="sample1_8h_html_afcac3511429181907b324fa566e6147da6d8bd6a7b94429614d045d1cb574d7ef"><div class="ttname"><a href="sample1_8h.html#afcac3511429181907b324fa566e6147da6d8bd6a7b94429614d045d1cb574d7ef">Enum3</a></div><div class="ttdoc">This is Enum3. </div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00012">sample1.h:12</a></div></div>
<div class="ttc" id="sample1_8h_html_a2b69c74036cc3a2a35c5aac054dfa154"><div class="ttname"><a href="sample1_8h.html#a2b69c74036cc3a2a35c5aac054dfa154">CClass</a></div><div class="ttdeci">CClass</div><div class="ttdoc">This is CClass. </div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00016">sample1.h:16</a></div></div>
<div class="ttc" id="sample1_8h_html_afcac3511429181907b324fa566e6147dab835515b078d0fde7ddfb245937aa0aa"><div class="ttname"><a href="sample1_8h.html#afcac3511429181907b324fa566e6147dab835515b078d0fde7ddfb245937aa0aa">Enum2</a></div><div class="ttdoc">This is Enum2. </div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00011">sample1.h:11</a></div></div>
<div class="ttc" id="sample1_8h_html_afcac3511429181907b324fa566e6147da8ac106a2a0779565ad972ac4314f4d10"><div class="ttname"><a href="sample1_8h.html#afcac3511429181907b324fa566e6147da8ac106a2a0779565ad972ac4314f4d10">Enum1</a></div><div class="ttdoc">This is Enum1. </div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00010">sample1.h:10</a></div></div>
<div class="ttc" id="sample1_8h_html_afcac3511429181907b324fa566e6147d"><div class="ttname"><a href="sample1_8h.html#afcac3511429181907b324fa566e6147d">SomeEnum</a></div><div class="ttdeci">SomeEnum</div><div class="ttdef"><b>Definition:</b> <a href="sample1_8h_source.html#l00009">sample1.h:9</a></div></div>
</div><!-- fragment --></div><!-- contents -->
</div><!-- doc-content -->
<!-- start footer part -->
<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
  <ul>
    <li class="navelem"><a class="el" href="dir_c807abc01c3358b2409f070c37c24516.html">folder</a></li><li class="navelem"><a class="el" href="sample1_8h.html">sample1.h</a></li>
    <li class="footer">Generated on Mon Mar 31 2014 11:18:48 for Sample Project by
    <a href="http://www.doxygen.org/index.html">
    <img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.8.6 </li>
  </ul>
</div>
</body>
</html>
